g_docs.go

```diff
@@ -770,6 +770,30 @@ func parseObject(d *ast.Object, k string, m *swagger.Schema, realTypes *[]string
 
 				var tagValues []string
 				stag := reflect.StructTag(strings.Trim(field.Tag.Value, "`"))
+				
+				defaultValue := stag.Get("doc")
+				if defaultValue != ""{
+					r, _ := regexp.Compile(`default\((.*)\)`)
+					if r.MatchString(defaultValue) {
+						res := r.FindStringSubmatch(defaultValue)
+						mp.Default = res[1]
+						switch realType{
+							case "int","int64", "int32", "int16", "int8":
+								mp.Default, _ = strconv.Atoi(res[1])
```

Comment:
@astaxie Yes you are right. So you can not leave. That's what you do or do I add?